File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        j = 0
        for i in range(len(nums)-1):
            for j in range(len(nums)):
                if (nums[i] + nums[j] == target) and (i != j):
                    return [i,j]        
        return [i, j]

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        ios_base::sync_with_stdio(0);
        cin.tie(0);
        cout.tie(0);
        unordered_map<int, int> m;
        for(int i  = 0 ; i < nums.size() ; i++)
        {
            if(m.find(target-nums[i]) == m.end())
                m[nums[i]] = i;
            else return {m[target-nums[i]], i};
        }
        return {-1,-1};
    }
};